src/selectors/url-state.js

```diff
@@ -255,3 +256,29 @@ export const getCommittedRangeLabels: Selector<string[]> = createSelector(
   getAllCommittedRanges,
   CommittedRanges.getCommittedRangeLabels
 );
+
+export const getSymbolStoreUrl: Selector<string> = createSelector(
+  getUrlState,
+  ({ symbolServerUrl }) => {
+    if (!symbolServerUrl) {
+      return SYMBOL_STORE_URL;
+    }
+    const allowedHostnames = new Set(['localhost', 'symbols.mozilla.org']);
```

Comment:
optional nit: because there's only 2 items in this array, this is very probably faster (and simpler) to use the array directly, with `.includes`. Creating a Set has some fixed CPU and memory cost that aren't worth it here.

Reply:
Also I wonder if we shouldn't check `https` and ports for everything that's not `localhost`. Therefore we would have a special case for `localhost` (`http` or `https` permitted, any port allowed), but for other hostnames we would only accept `https` on the standard port.

I'll keep it optional but I think this would be a better option and the code to implement this doesn't seem more complicated.